lib/ansible/modules/identity/keycloak/keycloak_client.py

```diff
@@ -39,57 +39,67 @@
 options:
     state:
         description:
-            - State of the client
+            - State of the client.
             - On C(present), the client will be created (or updated if it exists already).
-            - On C(absent), the client will be removed if it exists
-        choices: ['present', 'absent']
-        default: 'present'
+            - On C(absent), the client will be removed if it exists.
+        type: str
```

Comment:
"str" is always the default and was intentionally omitted to keep in line with other modules in the repository; has this policy been changed so that the type has to be specified even if left at default, now? (same comment for all the 'type: str additions ;)

Reply:
Yes, we need to add for the documentation. Because nobody before documented the types (it wasn't even allowed) we don't know if an omission means a string or not.

BTW I proposed a PR to get this directly from the arg_spec but this one was rejected. Which means we have to document it explicitly.
BTW We also recommend to add type-information to the arg_spec for clarity, but `required: no` or `default: None` should not be added. Only type-information.

Reply:
Example: https://docs.ansible.com/ansible/latest/modules/keycloak_client_module.html